### PR TITLE
feat: Add searchbar in manual

### PR DESCRIPTION
**Pull request by netchampfaris**
image

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/index.html [hunk 1]
```diff
@@ -13,6 +13,17 @@
 		</nav>
 		<h1>User Manual (English)</h1>
 		<p class='lead mx-0'>Explore modules to delve deep into ERPNext features</p>
+		<form action="/search">
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/index.html]: @netchampfaris, kindly add the searchbar in all md pages of docs, will give better accessibility

**Comment by vrms** [comment]
this is a must-have feature (actually not a new one. A search existed in the Manual before as in any other manual).

> @prasadarr: _kindly add the searchbar in all md pages of docs, will give better accessibility_

this may be of additional value but should not stop merging the PR as is (assuming that it works).
can we move that feature request of yours ("add the searchbar in all md pages") to a new issue and proceed with the merging of this PR?

**Comment by vrms** [comment]
great, thanks

**Comment by vrms** [comment]
created a new issue as per your comments https://github.com/frappe/erpnext_com/issues/770

**Pull request merged**